daemon/buildinfo.c

```diff
@@ -181,6 +181,14 @@ void print_build_info(void) {
     printf("    dbengine:                %s\n", FEAT_DBENGINE);
     printf("    Native HTTPS:            %s\n", FEAT_NATIVE_HTTPS);
     printf("    Netdata Cloud:           %s\n", FEAT_CLOUD);
+#ifndef ENABLE_ACLK
+#ifdef DISABLE_CLOUD
+    printf("        Cloud Disabled By User Request (e.g. installer flag --disable-cloud)\n");
+#else
+    printf("        Usable Libwebsockets:       %s\n", FEAT_LWS);
+    printf("        Usable Mosquitto:           %s\n", FEAT_MOSQUITTO);
```

Comment:
Strictly speaking, we shouldn't need this block here, as we have both LWS and mosquitto listed in the libaries.

I do agree that we should have something special to indicate that the user explicitly disabled cloud support though.

Reply:
yep you are right, I wanted to have it in one place but I can remove

Reply:
I tried to keep it together also in configure script:
so you have all you need near the line that says `can't build ACLK` and no need to scroll and guess..
but you are right this is not user facing thing but more of a troubleshooting so I will remove it

Reply:
changed it to your wishes